assets/ios/swift/Icons.xcassets/iconThread.imageset/iconThread.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="none" viewBox="0 0 18 18"><path fill="currentColor" d="M5 5.25a.75.75 0 0 0 0 1.5h8a.75.75 0 0 0 0-1.5H5ZM5 8.25a.75.75 0 0 0 0 1.5h4a.75.75 0 1 0 0-1.5H5Z"/><path fill="currentColor" fill-rule="evenodd" d="M3 .25A2.75 2.75 0 0 0 .25 3v14a.75.75 0 0 0 1.2.6L4.916 15c.217-.162.48-.25.75-.25H15A2.75 2.75 0 0 0 17.75 12V3A2.75 2.75 0 0 0 15 .25H3ZM1.75 3c0-.69.56-1.25 1.25-1.25h12c.69 0 1.25.56 1.25 1.25v9c0 .69-.56 1.25-1.25 1.25H5.666a2.75 2.75 0 0 0-1.65.55L1.75 15.5V3Z" clip-rule="evenodd"/></svg>
```

Comment:
There's something wrong with this file format that Xcode doesn't like. When I preview it, the icon shows up as blank:

![Screenshot 2023-03-02 at 12 00 35 pm](https://user-images.githubusercontent.com/6060466/222422973-308cb771-75e3-48df-91a8-b05352e77c5d.png)

Re-exporting the SVG in Affinity I get an output that Xcode is happy with:
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg width="100%" height="100%" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
    <path d="M5,5.25C4.589,5.25 4.25,5.589 4.25,6C4.25,6.411 4.589,6.75 5,6.75L13,6.75C13.411,6.75 13.75,6.411 13.75,6C13.75,5.589 13.411,5.25 13,5.25L5,5.25ZM5,8.25C4.589,8.25 4.25,8.589 4.25,9C4.25,9.411 4.589,9.75 5,9.75L9,9.75C9.411,9.75 9.75,9.411 9.75,9C9.75,8.589 9.411,8.25 9,8.25L5,8.25Z" style="fill-rule:nonzero;"/>
    <path d="M3,0.25C1.491,0.25 0.25,1.491 0.25,3L0.25,17C0.25,17.411 0.589,17.75 1,17.75C1.162,17.75 1.32,17.697 1.45,17.6L4.916,15C5.133,14.838 5.396,14.75 5.666,14.75L15,14.75C16.509,14.75 17.75,13.509 17.75,12L17.75,3C17.75,1.491 16.509,0.25 15,0.25L3,0.25ZM1.75,3C1.75,2.31 2.31,1.75 3,1.75L15,1.75C15.69,1.75 16.25,2.31 16.25,3L16.25,12C16.25,12.69 15.69,13.25 15,13.25L5.666,13.25C5.071,13.25 4.492,13.443 4.016,13.8L1.75,15.5L1.75,3Z"/>
</svg>
```

Reply:
That is because the SVG is using `currentColor` and XCode is not replacing that. I've changed the SVG fill color to be black so the icons can be previewed. 
If that is causing an issue and they can't be tinted, let me know!

Reply:
Ah ha! Works great now, we can easily tint as they're templated 👍